src/ARCTokenization/TopLevelParsers.fs

```diff
@@ -355,4 +384,26 @@ type Assay =
                     |> Tokenization.ARCtrl.ARCTable.tokenizeColumns
                 )
             )
-            |> Map.ofSeq
\ No newline at end of file
+            |> Map.ofSeq
+
+    /// <summary>
+    /// Returns an annotation tables from an IParam if the given token contains a filepath with the standard assay file name ("isa.assay.xlsx").
+    /// Map of string * `IParam` 2D List representing the individual parts parts of the Process graph, 
+    /// where the string is the name of the worksheet that contained the table, 
+    /// and the 2D lists represent a single table in which the inner 1D lists represent a single column.
+    /// </summary>
+    /// <param name="rootPath">ARC root path</param>
+    /// <param name="refFileToken">IParam of the ARC Tokens</param>
+    static member parseProcessGraphColumnsFromToken (rootPath:string) (refFileToken: IParam) =
+        ISA.parseProcessGraphColumnsFromToken rootPath refFileToken
+
+    /// <summary>
+    /// Returns a seq of annotation tables from an IParam seq if the given tokens contains a filepath with the standard assay file name ("isa.assay.xlsx").
+    /// Map of string * `IParam` 2D List representing the individual parts parts of the Process graph, 
+    /// where the string is the name of the worksheet that contained the table, 
+    /// and the 2D lists represent a single table in which the inner 1D lists represent a single column.
+    /// </summary>
+    /// <param name="rootPath">ARC root path</param>
+    /// <param name="refFileToken">IParam seq of the ARC Tokens</param>
+    static member parseProcessGraphColumnsFromTokens (rootPath:string) (refFileTokens: #IParam seq) =
```

Comment:
The xml comments still do not help that much. 

> IParam seq of the ARC Tokens

says nothing to me. 

Also 

> Returns a seq of annotation tables from an IParam seq if the given tokens contains a filepath with the standard assay file name ("isa.assay.xlsx") 

is only technically true, as we now match the term. So i would suggest something like this:

> Returns a seq of annotation tables from an IParam seq for each contained token that is annotated with the term 'Assay File' (e.g, files in the ARC named `isa.assay.xlsx`).

and for refFileToken**s** (typo there as well)

> a seq of IParams that may contain relevant tokens